config/linux-kgpe-d16_server.config

```diff
@@ -0,0 +1,326 @@
+CONFIG_LOCALVERSION="-heads"
```

Comment:
cp config/linux.. to build/linux/..
cd build/linux..
make savedefconfig
cp defconfig ../../config/linux..

for all linux configs

Reply:
Fixed with 2de4e7a4a5e31b6ea2486564de776fb731e89731